Config.h.in

```diff
@@ -0,0 +1,15 @@
+#ifndef CONFIG_H
+#define CONFIG_H
+
+#define SWIFT_LIB_DIR "@SWIFT_BINARY_DIR@\\lib\\swift"
+#define SWIFT_WINDOWS_LIB_DIR SWIFT_LIB_DIR "\\windows"
+#define SWIFT_BUILTIN_MODULE_PATH SWIFT_WINDOWS_LIB_DIR "\\x86_64"
+#define SWIFT_CLANG_RESOURCE_DIR SWIFT_LIB_DIR "\\clang"
+#define SWIFT_SHIMS_RESOURCE_DIR SWIFT_LIB_DIR "\\shims"
+
+#cmakedefine DEFAULT_MODULE_CACHE_PATH "@DEFAULT_MODULE_CACHE_PATH@"
+#ifndef DEFAULT_MODULE_CACHE_PATH
+    #define DEFAULT_MODULE_CACHE_PATH "C:\\Windows\\Temp\\SwiftModuleCache"
+#endif
```

Comment:
This really is not a good idea.  The paths at build may not match the paths at runtime.

Reply:
For the cache path? What should I set it to?

Reply:
You could have it be some default value and allow people to set it with a command line argument?

Reply:
Ok, that sounds reasonable